doc/ref/api/hvplot_api/hvplot.hvPlot.scatter.ipynb

```diff
@@ -19,9 +18,41 @@
     ".. backend-styling-options:: scatter\n",
     "```\n",
     "\n",
-    "## Examples\n",
+    "## Examples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Scatter plot with `marker` type changed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.earthquakes(\"pandas\")\n",
     "\n",
-    "TBD"
+    "df.hvplot.scatter(\n",
```

Comment:
It seems to me this is a bit of a too "complicated" example (tiles) for the first one on this page.

Reply:
OK. Will change it.